pkg/apis/kops/cluster.go

```diff
@@ -486,6 +486,11 @@ func (c *Cluster) FillDefaults() error {
 		c.Spec.MasterPublicName = "api." + c.ObjectMeta.Name
 	}
 
+	if c.Spec.EncryptionConfig == nil {
```

Comment:
Let's have a comment here that this is temporary for the purposes of getting the e2e test to pass